calico/src/main/scala/calico/html/Html.scala

```diff
@@ -41,6 +42,8 @@ sealed trait Html[F[_]](using F: Async[F])
       KeyedChildrenModifiers[F],
       HtmlAttrModifiers[F]:
 
+  given Dom[F] = Dom.forAsync
```

Comment:
So when you do `import html.{*, given}` you get a `Dom[F]` without having to add it as an extra constraint.